src/VisualStudio/Core/Def/EditorConfigSettings/SettingsEditorFactory.cs

```diff
@@ -175,5 +175,17 @@ public int MapLogicalView(ref Guid rguidLogicalView, out string? pbstrPhysicalVi
                 return VSConstants.E_NOTIMPL;   // you must return E_NOTIMPL for any unrecognized rguidLogicalView values
             }
         }
+
+        public object? GetDocumentData(uint grfCreate, string pszMkDocument, IVsHierarchy pHier, uint itemid)
+            => null;
+
+        public object? GetDocumentView(uint grfCreate, string pszPhysicalView, IVsHierarchy pHier, IntPtr punkDocData, uint itemid)
+            => null;
+
+        public string? GetEditorCaption(string pszMkDocument, string pszPhysicalView, IVsHierarchy pHier, IntPtr punkDocData, out Guid pguidCmdUI)
+            => throw new NotImplementedException();
```

Comment:
We don't need to implement these?

Reply:
I can't find any internal use of this method at all.

Reply:
So our implementation for our [AbstractEditorFactory](https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs) looks like it has an implementation for the first, but not the latter two.

Reply:
Testing it out locally I could not find a way to get `GetEditorCaption` to be called.